src/jobflow_remote/cli/types.py

```diff
@@ -62,7 +62,7 @@
     typer.Option(
         "--flow-id",
         "-fid",
-        help="One or more flow ids",
+        help="One or more flow ids. Can the db id (i.e. an integer) or a string (i.e. the uuid)",
```

Comment:
The db id can also be an integer with a prefix (if you set db_id_prefix in the queue settings). I'd just put that it can be the db id or uuid. If you have another suggestion, go ahead.
```suggestion
        help="One or more flow ids. Each flow id can be the db id or a uuid",
```

Reply:
I have copied this from other type definitions. Originally the db_id was only an integer, but at some point you allowed a prefix. Probably worth to update all of the documentation.

Reply:
True, we should at some point make a check of the whole doc. I will resolve this now.